scipy/stats/_boost/include/_info.py

```diff
@@ -13,6 +13,7 @@ class _KlassMap(NamedTuple):
     'binomial': _KlassMap('binom', ('n', 'p')),
     'negative_binomial': _KlassMap('nbinom', ('n', 'p')),
     'hypergeometric': _KlassMap('hypergeom', ('r', 'n', 'N')),
+    'non_central_f': _KlassMap('ncf', ('dfn', 'dfd', 'nc')),
```

Comment:
Add the Boost `non_central_f` distribution to the list of classes that ufuncs will be generated for.

I found the name of the Boost class (key in the `_KlassMap` dictionary entry) from the name of the [Boost header file](https://www.boost.org/doc/libs/1_78_0/libs/math/doc/html/math_toolkit/dist_ref/dists/nc_f_dist.html) and the SciPy name and constructor arguments (value in the `KlassMap` dictionary entry) from the [SciPy documentation](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.ncf.html)